cert_manager/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.12.0",
+    "setuptools<61",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-cert-manager"
+description = "The cert-manager check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "cert_manager",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: System :: Monitoring",
+]
+dependencies = [
+    "datadog-checks-base>=4.2.0",
```

Comment:
@sarah-witt @hithwen I've noticed that the version required here is much older than in the rest of integrations. Should we keep this as it was or use a different version?

Reply:
We should update the version to what other openmetrics checks use